app/src/main/java/org/greenstand/android/TreeTracker/view/TextButton.kt

```diff
@@ -86,6 +86,42 @@ fun BoxScope.ArrowButton(
     }
 }
 
+@Composable
+fun AcceptButton(
```

Comment:
Let's try to have less repeated code here. The contents of Accept and Decline are the same aside from color and icon. Can we make a generic button that accepts those two params? Then these two buttons can just reuse that button with different icon/colors.

This will allow the buttons to stay in sync.

Currently we need to keep two buttons in sync. Someone can update one and forget about the other.
If they both reuse a single button, they will always stay in sync.